cli/src/semgrep/autofix.py

```diff
@@ -130,53 +134,122 @@ def _write_contents(path: Path, contents: str) -> None:
     path.write_text(contents)
 
 
+def matches_compare(x: RuleMatch, y: RuleMatch) -> int:
+    # If paths are not the same, just order based on that.
+    # I just need a total ordering on matches.
+    if x.path < y.path:
```

Comment:
I sometimes will go instead for
```
if x.path != y.path: 
    return x.path - y.path 
else:
    <next case:>
```
but I'm fine with this too. Mostly just mentioning it if you hadn't considered it